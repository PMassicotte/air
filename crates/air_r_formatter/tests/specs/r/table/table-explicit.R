#' [format]
#' table = ["foo"]

foo(
~x,~y,
1,2,
3,4
)

a <- foo(
~x,~y,
1,2,
3,4
)

bar(
~x,~y,
1,2,
3,4
)

# Should format as table by default
tribble(
~x,~y,
1,2,
3,4
)

# Should format as table by default
fcase(
x<5L,1L,
x>5L,3L,
default=5L
)
