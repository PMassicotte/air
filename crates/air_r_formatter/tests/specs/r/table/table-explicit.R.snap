---
source: crates/air_formatter_test/src/snapshot_builder.rs
info: r/table/table-explicit.R
---
# Input

```R
#' [format]
#' table = ["foo"]

foo(
~x,~y,
1,2,
3,4
)

a <- foo(
~x,~y,
1,2,
3,4
)

bar(
~x,~y,
1,2,
3,4
)

# Should format as table by default
tribble(
~x,~y,
1,2,
3,4
)

# Should format as table by default
fcase(
x<5L,1L,
x>5L,3L,
default=5L
)

```


=============================

# Outputs

## Output 1

-----
Indent style: Space
Indent width: 2
Line ending: LF
Line width: 80
Persistent line breaks: Respect
Table: fcase, foo, tribble
-----

```R
#' [format]
#' table = ["foo"]

foo(
  ~x , ~y ,
   1 ,  2 ,
   3 ,  4
)

a <- foo(
  ~x , ~y ,
   1 ,  2 ,
   3 ,  4
)

bar(
  ~x,
  ~y,
  1,
  2,
  3,
  4
)

# Should format as table by default
tribble(
  ~x , ~y ,
   1 ,  2 ,
   3 ,  4
)

# Should format as table by default
fcase(
  x < 5L , 1L ,
  x > 5L , 3L ,
  default = 5L
)
```
